code/datums/mind.dm

```diff
@@ -83,6 +83,10 @@
 	if(new_character.mind)								//disassociate any mind currently in our new body's mind variable
 		new_character.mind.current = null
 
+	if(istype(current) && islist(current.antag_datums)) //wow apparently current isn't always living good fucking job SOMEONE

```

Comment:
Apparently new_player mobs have a mind because some reason, so current, typed to `/mob/living`, isn't always a living mob.